### PR TITLE
Update Institution fuzzy-search order by

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
Update Institution "fuzzy search" order to increase relevance of top results by weighting institutions by the number of GI Bill students.

[ZenHub Issue #11197](https://github.com/department-of-veterans-affairs/va.gov-team/issues/11197#event-3548567034)

## Testing done
Tested locally and QA reviewed

## Screenshots
N/A

## Acceptance criteria
- [x] Institution search results return results sorted by number of GI Bill students descending then similarity on institution name.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

File: app/controllers/v0/institutions_controller.rb [hunk 1]
```diff
@@ -30,10 +30,7 @@ def index
       }
 
       if @query.key?(:fuzzy_search)
-        order_by = <<-SQL
-          (SIMILARITY(institution, :search_term) + SIMILARITY(city, :search_term)) / 2 DESC,
-          institution ASC
-        SQL
+        order_by = 'SIMILARITY(institution, :search_term) * COALESCE(gibill, 0) DESC, institution'
```
Comment (LindseySaari) [inline app/controllers/v0/institutions_controller.rb:33]: Just curious as to why the SIMILARITY on city is no longer needed

Comment (dmccurdybah) [reply app/controllers/v0/institutions_controller.rb:33]: Longer answer: This is getting in front of users for testing and that's driving us to try different things until they think the "feel" is right.

Short answer: Tried it, didn't like it.

Comment (LindseySaari) [reply app/controllers/v0/institutions_controller.rb:33]: 👍

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**